src/com/komiamiko/fcorbit/Main.java

```diff
@@ -207,7 +212,11 @@ public void mouseExited(MouseEvent e) {
 
 			@Override
 			public void mouseDragged(MouseEvent e) {
-				graphicEditor.mouseDragged = true;
+				if(!graphicEditor.mouseDragged && Math.hypot(
+						e.getX() - graphicEditor.originMousex,
+						e.getY() - graphicEditor.originMousey) >= MOUSE_DRAG_MIN_PIXELS) {
```

Comment:
This reflects the static field addition of MOUSE_DRAG_MIN_PIXELS in line 40 and it makes sure that MOUSE_DRAG_MIN_PIXELS is reached before moving the cursor.

Reply:
Yes, that's the point.